doc/biocpp/presentation/index.html

```diff
@@ -867,17 +867,17 @@ <h3>Schedule<small>*</small>:</h3>
 <textarea data-template>
 ###### Working with views
 
-Learning objectives
+Learning objectives:
 <!-- .element: class="fragment" -->
 
 * <!-- .element: class="fragment" -->
-  what is a view
+  What is a view.
 * <!-- .element: class="fragment" -->
-  how to chain views
+  How to chain views.
 * <!-- .element: class="fragment" -->
-  writing generic algorithms
+  Writing generic algorithms.
 * <!-- .element: class="fragment" -->
-  understanding and using concepts in generic code
+  Understanding and using concepts in generic code.
 
 </textarea>
 </section>
```

Comment:
A few lines beneath:

`Write code that checks if there is any`

What that's supposed to mean?